### PR TITLE
Failed to scale down tikv instances less than 3

### PR DESCRIPTION
**Pull request by wxbty**
### What problem does this PR solve?
Misleading users when scaling down tikv
  
In the k8s test environment, after deploying tidb, the number of instances of tipd, tidb, and tikv in my cluster is 3, 3, and 3, respectively.
I tried to use the kubectl command or update tidb-cluster.yaml to shrink the cluster to 1, 1, and 3, and the results showed that the pod was successful.

I used the following command or apply tidb-cluster.yaml
kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"tipd":{"replicas":1}}}'
kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"tidb":{"replicas":1}}}'

However, when I try to scale down tikv, there is no change in the pod, there are always 3, and the status is running. I tried again and again, the problem is still there, I think there is something wrong.
This pr has the same problem, but no reply #4546 

I checked the pod logs
tidb-controller-manager output the following logs. which is same as 4546

can't scale in TiKV of TidbCluster [tidb-cluster/basicai], cause the number of up stores is equal to MaxReplicas in PD configuration(3), and the store in Pod basicai-tikv-2 which is going to be deleted is up too

In the [documentation](https://docs.pingcap.com/tidb-in-kubernetes/dev/scale-a-tidb-cluster), found this description 
According to the prompt, I need to modify the value of pd MaxReplicas.
But it's hard to find how to configure MaxReplicas for pd
There is no [configuration document](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md#tidbspec) for k8s

Finally found that you can use pd-ctl (do not want to install it separately) or sql command line real-time change to do it
[link](https://docs.pingcap.com/zh/tidb/stable/dynamic-config)

This problem has been bothering me for a long time. I just want to scale down tikv in test environment

### What is changed and how does it work?

It seems that this question will mislead newbies, so I propose two solutions to solve.
1. Add the MaxReplicas field in the pd area of tidb-cluster.yaml, so that it can be changed by kubectl patch ... spec, and the api field adds a description of the limit between fields. If this is allowed, I will mention another pr to solve.
2. If only online changes are allowed due to design or other reasons, then at least the error message is need more detailed, such as the current pr submission

### Code changes

- [ yes ] Has Go code change
- [ no ] Has CI related scripts change

### Tests

Only the prompt has been modified, no test is needed for the time being

### Side effects

no

### Related changes

- [ no ] Need to cherry-pick to the release branch 
- [yes ] Need to update the documentation  https://docs.pingcap.com/tidb-in-kubernetes/dev/scale-a-tidb-cluster
       Tell the user how to configure  MaxReplicas
       【original】
       Note 
      When the number of UP stores is equal to or less than the parameter value of MaxReplicas in the PD configuration, the TiKV components can not be scaled in.  

### Release Notes
no

```release-note

```

### CODE REVIEW

**Comment by x-shadow-man** [comment]
> /cc @AstroProfundis @tennix

@AstroProfundis @tennix @wxbty. Thx for your time. Pls feel free to check the PR. Thx a lot. :-)

**Review by csuzhangxc — APPROVED** [review]

File: pkg/manager/member/tikv_scaler.go [hunk 1]
```diff
@@ -305,7 +305,7 @@ func (s *tikvScaler) preCheckUpStores(tc *v1alpha1.TidbCluster, podName string,
 		return false
 	} else if upNumber == maxReplicas {
 		if storeState == v1alpha1.TiKVStateUp {
-			errMsg := fmt.Sprintf("can't scale in TiKV of TidbCluster [%s/%s], cause the number of up stores is equal to MaxReplicas in PD configuration(%d), and the store in Pod %s which is going to be deleted is up too", tc.GetNamespace(), tc.GetName(), maxReplicas, podName)
+			errMsg := fmt.Sprintf("can't scale in TiKV of TidbCluster [%s/%s], cause the number of up stores is equal to MaxReplicas in PD configuration(%d), and the store in Pod %s which is going to be deleted is up too. MaxReplicas can be update online using pd-ctl or SQL statements ,as https://docs.pingcap.com/tidb/stable/dynamic-config", tc.GetNamespace(), tc.GetName(), maxReplicas, podName)
```
Comment (csuzhangxc) [inline pkg/manager/member/tikv_scaler.go:308]:
```suggestion
			errMsg := fmt.Sprintf("can't scale in TiKV of TidbCluster [%s/%s], cause the number of up stores is equal to MaxReplicas in PD configuration(%d), and the store in Pod %s which is going to be deleted is up too. MaxReplicas can be update online using pd-ctl or SQL statements, as https://docs.pingcap.com/tidb/stable/dynamic-config", tc.GetNamespace(), tc.GetName(), maxReplicas, podName)
```

**Review by KanShiori — APPROVED** [review]

**Pull request merged**